website/addons/base/views.py

```diff
@@ -202,7 +204,27 @@ def create_waterbutler_log(payload, **kwargs):
     auth = Auth(user=user)
     node = kwargs['node'] or kwargs['project']
 
-    if action in (NodeLog.FILE_MOVED, NodeLog.FILE_COPIED):
+    try:
+        dest = payload['destination']
+        src = payload['source']
+
+        if src is not None and dest is not None:
+            dest_path = dest['materialized']
+            src_path = src['materialized']
+            if str(dest_path)[-1] == "/" and str(src_path)[-1] == "/":
+                dest_path = os.path.dirname(dest_path)
+                src_path = os.path.dirname(src_path)
+            if os.path.split(dest_path)[0] == os.path.split(src_path)[0]:
+                if dest['provider'] == src['provider']:
+                    if dest['nid'] == src['nid']:
+                        if dest['name'] == src['name']:
+                            return {'status': 'success'}
```

Comment:
@chennan47 mentioned that there should be no log created when a user tries to rename a folder with a slash at the end: i.e. folder to folder/, the / is just ignored.